site/content/docs/user/known-issues.md

```diff
@@ -234,6 +235,19 @@ an 03 17:42:41 kind-1-control-plane kubelet[3804]: F0103 17:42:41.470269 3804 ku
 
 This problem seems to be related to a [bug in Docker][moby#9939].
 
+## Docker permission denied
+
+When using `kind`, we assume that the user you are executing kind as has permission to use docker.
+If you initially ran Docker CLI commands using `sudo`, you may see the following error, which indicates that your `~/.docker/` directory was created with incorrect permissions due to the `sudo` commands.
+
+```
+WARNING: Error loading config file: /home/user/.docker/config.json
+open /home/user/.docker/config.json: permission denied
+```
+
+To fix this problem, either follow the docker's docs [manage docker as a non root user][manage docker as a non root user],
+or try to use `sudo` before your commands (if you get `command not found` please check [this comment about sudo with kind][sudo with kind]).
```

Comment:
I think this is fine for now, but we should probably iterate on this at some point, really this should be "run as a user with permission to docker (E.G. using `su`)":

Reply:
Got it, I will look for opportunities to update it later.